src/games/strategy/triplea/delegate/MustFightBattle.java

```diff
@@ -669,6 +669,13 @@ private void removeAirNoLongerInTerritory() {
     }
     return steps;
   }
+  
+  public boolean canAttackerRetreatSome() {
```

Comment:
public methods should have a javadoc.
What does "Some" mean? It's not very precise.